packages/flutter_tools/bin/fuchsia_builder.dart

```diff
@@ -56,8 +59,25 @@ Future<Null> main(List<String> args) async {
     packagesPath: argResults[_kOptionPackages],
     includeRobotoFonts: true,
   );
-  if (result != 0) {
-    printError('Error building $outputPath: $result.');
+  if (buildResult != 0) {
+    printError('Error building $outputPath: $buildResult.');
+    exit(buildResult);
+  }
+  final int headerResult = _addHeader(outputPath, argResults[_kOptionHeader]);
+  if (headerResult != 0) {
+    printError('Error adding header to $outputPath: $headerResult.');
+  }
+  exit(headerResult);
+}
+
+int _addHeader(String outputPath, String header) {
+  try {
+    final File outputFile = new File(outputPath);
+    final List<int> content = outputFile.readAsBytesSync();
+    outputFile.writeAsStringSync('$header\n');
+    outputFile.writeAsBytesSync(content, mode: FileMode.APPEND);
```

Comment:
This is sort of lame.  Can we not write this information into the file when we first create it?